docs/core/porting/project-structure.md

```diff
@@ -0,0 +1,79 @@
+# Organizing Your Project to Support .NET Framework and .NET Core
+
+This article is to help project owners who want to compile their solution against .NET Framework and .NET Code side-by-side.  It provides several options to organize projects to help developers achieve this goal.
+
+## General Guidance
+
+1.  Create an xproj that targets multiple frameworks.
+    * Scenario is described [here](#replace-existing-projects-with-a-multi-targeted-net-core-project-xproj)
+2.  For projects that use complex logic in their build process, create a Portable Class Library (PCL) that targets .NET Core.
+    * Scenario is described [here](#create-a-portable-class-library-pcl-to-target-net-core)
+
+These are general rules for organizing your project. The scenarios below outline a few variations that can be made based on what fits your repository best.
+
+## Example
+
+Consider the repository below:
+
+![][example-initial-project]
+
+[**Source Code**][example-initial-project-code]
+
+There are several different ways to add support for .NET Core for this repository depending on the constraints and complexity of existing projects which are described below.
+
+### Replace Existing Projects with a Multi-targeted .NET Core Project (xproj)
```

Comment:
Is this and the following sections really a subsection of Example? The way docs work it will add a mini-toc at the top with your H2 headers that will read:
- General Guidance
- Example